pkg/poolcache/poolcache.go

```diff
@@ -89,6 +90,19 @@ func (c *Cache) service() {
 		req := <-c.requestChannel
 		resp := &response{}
 		switch req.requestType {
+		case getValueByFunctionAndAndress:
+			otelUtils.LoggerWithTraceID(req.ctx, c.logger).Debug("Get value By Function And Address", zap.String("function", req.function.(string)), zap.String("address", req.address.(string)))
```

Comment:
Fix letter casing in the message: "Get value by function and address", also consider reducing whitespace here